src/qt/locale/bitcoin_da.ts

```diff
@@ -3,4349 +3,286 @@
     <name>AddressBookPage</name>
     <message>
         <source>Right-click to edit address or label</source>
-        <translation type="unfinished">Højreklik for at redigere adresse eller etiket</translation>
-    </message>
-    <message>
-        <source>Create a new address</source>
-        <translation type="unfinished">Opret en ny adresse</translation>
-    </message>
-    <message>
-        <source>&amp;New</source>
-        <translation type="unfinished">&amp;Ny</translation>
+        <translation type="unfinished">￼  Về     Tải về     blog     phát hành     Phát triển     Liên hệ                                Tiếng Anh                                                                                                    Tiếng Tây Ban Nha                                                  Nhật Bản                                                  Tiếng Trung giản thể                                                  繁體中文                                                                            Phát hành  Bitcoin Core 28.0  Bitcoin Core 27.1  Bitcoin Core 27.0  Bitcoin Core 26.2  Bitcoin Core 26.1  Bitcoin Core 26.0  Bitcoin Core 25.2  Bitcoin Core 25.1  Bitcoin Core 25.0  Bitcoin Core 24.2  Bitcoin Core 24.1  Lõi Bitcoin 24.0.1  Bitcoin Core 23.2  Bitcoin Core 23.1  Bitcoin Core 23.0  Bitcoin Core 22.1  Bitcoin Core 22.0  Bitcoin Core 0.21.2  Bitcoin Core 0.21.1  Bitcoin Core 0.21.0  Bitcoin Core 0.20.2  Bitcoin Core 0.20.1  Bitcoin Core 0.20.0  Bitcoin Core 0.19.1  Lõi Bitcoin 0.19.0.1  Bitcoin Core 0.18.1  Bitcoin Core 0.18.0  Lõi Bitcoin 0.17.1  Lõi Bitcoin 0.17.0.1  Bitcoin Core 0.17.0  Lõi Bitcoin 0.16.3  Lõi Bitcoin 0.16.2  Lõi Bitcoin 0.16.1  Bitcoin Core 0.16.0  Bitcoin Core 0.15.2  Lõi Bitcoin 0.15.1  Lõi Bitcoin 0.15.0.1  Bitcoin Core 0.15.0  Lõi Bitcoin 0.14.3  Bitcoin Core 0.14.2  Bitcoin Core 0.14.1  Bitcoin Core 0.14.0  Bitcoin Core 0.13.2  Lõi Bitcoin 10.11.12.8333  Bitcoin Core 0.13.0  Bitcoin Core 0.12.1  Bitcoin Core 0.12.0  Bitcoin Core 0.11.2  Bitcoin Core 1010568.11.1  Bitcoin Core 10.11.0..  Twitter Pháp lý | Chính sách bảo mật | RSS￼  ￼ © 2024 Lõi Bitcoin</translation>
```

Comment:
This looks like spam.

Reply:
Removed this one and other cases (bitcoin addresses) from this translation.